src/backends/ncnn/ncnnlib.cc

```diff
@@ -140,14 +152,21 @@ namespace dd
 		  blank_label = ad_output.get("blank_label").get<int>();
 	      }
 	  }
-	
+       bool timeserie = false;
+       if (typeid(inputc) == typeid(CSVTSNCNNInputFileConn))
```

Comment:
I believe this should be set once and for all, and `timeserie` be a class variable instead since the input connector cannot be changed on a living service.

Reply:
putting timeserie as a mltype is on my task list